internal/pkg/describe/errors.go

```diff
@@ -15,3 +20,14 @@ type ErrManifestNotFoundInTemplate struct {
 func (err *ErrManifestNotFoundInTemplate) Error() string {
 	return fmt.Sprintf("manifest metadata not found in template of stack %s-%s-%s", err.app, err.env, err.name)
 }
+
+type errLBWebSvcsOnCFWithoutAlias struct {
+	services   []string
+	aliasField string
+}
+
+// Error implements the error interface.
+func (err *errLBWebSvcsOnCFWithoutAlias) Error() string {
+	return fmt.Sprintf("%v %v must have %q specified when CloudFront is enabled", english.PluralWord(len(err.services), "service", "services"),
```

Comment:
nit: using "%s" instead of "%v"
```go

	return fmt.Sprintf("%s %s must have %q specified when CloudFront is enabled", english.PluralWord(len(err.services), "service", "services"),
		english.WordSeries(template.QuoteSliceFunc(err.services), "and"), err.aliasField)
```